messages/retrieve.metadata.preview.md

```diff
@@ -40,6 +40,10 @@ This flag applies only to orgs that allow source tracking. It has no effect on o
 
 Omit ignored files.
 
+# flags.concise.description
```

Comment:
FWIW, I think the summary was clear enough and having a huge help file means you gotta scroll back up to see the flags.  Will defer to @jshackell-sfdc.

Reply:
I can remove the description, but the other flags have descriptions.  The inconsistency won't be odd?